pyproject.toml

```diff
@@ -6,7 +6,7 @@ build-backend = "setuptools.build_meta"
 name = "rlbot"
 description = "A high performance Python interface for communicating with RLBot v5."
 dynamic = ["version"]
-requires-python = ">= 3.11"
+requires-python = ">= 3.12"
```

Comment:
What's up with this? This is another big breaking change - what language feature is now used in the interface that requires Python 3.12?

Reply:
My bad. Was Python 10 when I was initially cloned and brainfarted to think 11 didn't work either. Reverted.